test.js

```diff
@@ -270,3 +272,8 @@ test('handle undefined', t => {
 	const expected = undefined;
 	t.is(cleanStack(stack, {pretty: true}), expected);
 });
+
+test('exports for home-directory files match', t => {
```

Comment:
I don't really see the point of this test.

Reply:
Initially I didn't update `home-directory-browser.js` with the same change, which would've broken browser behavior. I figured it would be nice to have a test to help ensure the return type from both files is symmetrical.